config/config.sample.php

```diff
@@ -166,70 +170,74 @@
  * on. If no enabled apps are found it defaults to the Files app.
  */
 'defaultapp' => 'files',
 
 /**
- * `true` enables the Help menu item in the user menu (top right of the
- * ownCloud Web interface). `false` removes the help item.
+ * Enables or removes the Help menu item in the user menu
+ * The Help menue can be found in top right of the ownCloud Web interface.
+ * `true` enables the Help menu item, `false` removes the help item.
  */
 'knowledgebaseenabled' => true,
 
 /**
- * `true` enables avatars, or user profile photos. These appear on the User
- * page, on user's Personal pages and are used by some apps (contacts, mail,
- * etc). `false` disables them.
+ * Enables or disable  avatars or user profile photos
+ * `true` enables avatars, or user profile photos, `false` disables them.
+ * These appear on the User page, on user's Personal pages and are used by some apps
+ * (contacts, mail, etc). 
  */
 'enable_avatars' => true,
 
 /**
- * `true` allows users to change their display names (on their Personal
- * pages), and `false` prevents them from changing their display names.
+ * Allow or disallow users to change their display names
+ * `true` allows users to change their display names (on their Personal pages),
+ * `false` prevents them from changing their display names.
  */
 'allow_user_to_change_display_name' => true,
 
 /**
- * Lifetime of the remember login cookie, which is set when the user clicks the
- * `remember` checkbox on the login screen. The default is 15 days, expressed
- * in seconds.
+ * Define the lifetime of the remember login cookie
+ * The remember login cookie is set when the user clicks the `remember` checkbox
+ * on the login screen. The default is 15 days, expressed in seconds.
  */
 'remember_login_cookie_lifetime' => 60*60*24*15,
 
 /**
- * The lifetime of a session after inactivity; the default is 20 minutes,
- * expressed in seconds.
+ * Define the lifetime of a session after inactivity
+ * The default is 20 minutes, expressed in seconds.
  */
 'session_lifetime' => 60 * 20,
 
 /**
- * Enable or disable session keep-alive when a user is logged in to the Web UI.
+ * Enable or disable session keep-alive when a user is logged in to the Web UI
  * Enabling this sends a "heartbeat" to the server to keep it from timing out.
  */
 'session_keepalive' => true,
 
 /**
- * Enforces token only authentication for apps and clients connecting to ownCloud.
+ * Enforces token only authentication for apps and clients connecting to ownCloud
  * If enabled, all access requests using the users password are blocked for enhanced security.
  * Users have to generate special app-passwords (tokens) for their apps or clients in their personal
  * settings which are further used for app or client authentication. Browser logon is not affected.
  */
 'token_auth_enforced' => false,
 
 /**
+ * Define additional login buttons on the logon screen
```

Comment:
```suggestion
 * Defines additional login buttons on the logon screen
```

do we be consistent?